js/client-battle.js

```diff
@@ -11,7 +11,7 @@
 
 			this.isSideRoom = Tools.prefs('rightpanelbattles');
 
-			this.$el.addClass('ps-room-opaque').html('<div class="battle">Battle is here</div><div class="foehint"></div><div class="battle-log"></div><div class="battle-log-add">Connecting...</div><div class="battle-controls"></div><button class="battle-chat-toggle button" name="showChat"><i class="fa fa-caret-left"></i> Chat</button>');
+			this.$el.addClass('ps-room-opaque').attr('role', 'main').html('<div class="battle">Battle is here</div><div class="foehint"></div><div class="battle-log" role="complementary" aria-label="Battle Log"></div><div class="battle-log-add">Connecting...</div><div class="battle-controls" role="complementary" aria-label="Battle Controls"></div><button class="battle-chat-toggle button" name="showChat"><i class="fa fa-caret-left"></i> Chat</button>');
```

Comment:
`role="main"` is unfortunately a bad fit for battles, because you can have more than one battle open at a time.

The spec says:

> Within any document or application, the author SHOULD mark no more than one element with the main role.